src-gui/src/renderer/components/pages/help/SettingsBox.tsx

```diff
@@ -704,24 +704,32 @@ function NodeTable({
   );
 }
 
+const torForced = await getTorForcedExcuse();
```

Comment:
this can fail and is therefore not safe to call here

this should probably be stored in the Redux store (see `src-gui/src/store/features/rpcSlice.ts`)